test/sql-server-cdc/setup/setup.td

```diff
@@ -0,0 +1,58 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+$ sql-server-connect name=sql-server
+server=tcp:sql-server,1433;IntegratedSecurity=true;TrustServerCertificate=true;User ID=${arg.default-sql-server-user};Password=${arg.default-sql-server-password}
+
+$ sql-server-execute name=sql-server split-lines=false
+IF EXISTS (SELECT name FROM sys.databases WHERE name = N'test')
+BEGIN
+    ALTER DATABASE test SET SINGLE_USER WITH ROLLBACK IMMEDIATE;
+    DROP DATABASE test;
+END;
+
+$ sql-server-execute name=sql-server split-lines=false
+USE msdb;
+IF EXISTS (
+    SELECT 1
+    FROM msdb.dbo.sysjobs
+    WHERE name = N'DummyTicker'
+)
+BEGIN
+    EXEC sp_delete_job @job_name = N'DummyTicker';
+END;
+
+$ sql-server-execute name=sql-server
+CREATE DATABASE test COLLATE Latin1_General_100_CI_AI_SC_UTF8;
```

Comment:
Are we ok with making this the default collation? I had trouble altering the collation, it seems to mess with the dummy ticker job.

Reply:
also just noticed that, done. RFR